data_source.go

```diff
@@ -583,7 +584,7 @@ func (ds *AnySource) HandleExternalTriggers(externalTriggerRowcounts []int64) er
 		ds.writingState.externalTriggerFileBufferedWriter = bufio.NewWriter(ds.writingState.externalTriggerFile)
 		// write header
 		msg := fmt.Sprintf("# external trigger rowcounts as int64 binary data follows, "+
-			"rowcounts = framecounts*nrow+row (nrow=%4d)\n", ds.rowFrameRatio)
+			"rowcounts = framecounts*nrow+row (nrow=%4d)\n", ds.subframeDivisions)
```

Comment:
subframecounts = framecounts*subframerate+subframeoffset

??

Reply:
Yeah, why not?